bin/apisix

```diff
@@ -643,6 +643,34 @@ local function read_yaml_conf()
         merge_conf(default_conf, user_conf)
     end
 
+    -- check the Admin API token
+    if default_conf.apisix.enable_admin then
+        local help = [[
+ERROR: missing valid apisix.admin_key
+
+You can call `]] .. arg[0] .. [[ gen_admin_key` to generate a new Admin API key or
+manually update the `conf/config.yaml` file.
+]]
+        if type(default_conf.apisix.admin_key) ~= "table" or
+           #default_conf.apisix.admin_key == 0
+        then
+            io.stderr:write(help, "\n")
+            os.exit(1)
+        end
+
+        for _, admin in ipairs(default_conf.apisix.admin_key) do
+            if type(admin.key) == "table" then
+                admin.key = ""
+            else
+                admin.key = tostring(admin.key)
+            end
+
+            if admin.key == "" or admin.key:gsub("*", "") == "" then
```

Comment:
`*` is invalid, which is too shadowy

Reply:
In APISIX, the Admin API Key containing only `*` is invalid.
It is shadowy, but it works.

If we use `YOUR_OWN_API_TOKEN` as Key, then we have to disable it here too.